internal/component/database_observability/mysql/collector/query_sample.go

```diff
@@ -142,20 +143,20 @@ func (c *QuerySample) fetchQuerySamples(ctx context.Context) error {
 					sampleText = sampleText[:idx]
 				}
 			} else {
-				level.Debug(c.logger).Log("msg", "skipping parsing truncated query", "digest", digest)
+				level.Debug(c.logger).Log("msg", "skipping parsing truncated query", "digest", digest, "schema", schemaName)
 				continue
 			}
 		}
 
 		stmt, err := sqlparser.Parse(sampleText)
 		if err != nil {
-			level.Error(c.logger).Log("msg", "failed to parse sql query", "digest", digest, "err", err)
+			level.Error(c.logger).Log("msg", "failed to parse sql query", "digest", digest, "schema", schemaName, "err", err)
```

Comment:
Shall we use schemaName before digest. For consistency with other messages, and also to go from broad -> specific to help with log searching